activities/finance-management/authorization.md

```diff
@@ -0,0 +1,12 @@
+---
+type: Guide
+explains: how authorization is managed at the Foundation
+---
+
+# Authorization
+
+Two board members can approve payments above € 50.000. [See bylaws.](../../organization/articles-of-association.md)
```

Comment:
Suggest changing all periods in numbers in the thousands to commas, as is more common in English.